webapp/src/app.tsx

```diff
@@ -1665,6 +1665,14 @@ export class ProjectView extends data.Component<IAppProps, IAppState> {
         this.shareEditor.show(header);
     }
 
+    gettingStarted() {
+        pxt.tickEvent("btn.gettingstarted");
+        const targetTheme = pxt.appTarget.appTheme;
+        Util.assert(!this.state.sideDocsLoadUrl && targetTheme && targetTheme.gettingStarted != undefined);
```

Comment:
Without the != undefined, I get a type error: 
Argument of type 'string' is not assignable to parameter of type 'boolean'

I'd say it has something to do with the assert parameter type